scripts/build-docs.js

```diff
@@ -0,0 +1,155 @@
+/* eslint-env node */
```

Comment:
Thoughts on adding a shebang and making this script executable?  Of course, we could do this later when removing gulp or not do it at all if we want to ask users to call through npm run which makes sense as well.

Reply:
I just googled this and didn't even know it was a thing! I figured the only way to do it was an npm script. We can always come back and add it later if it's something we feel we need. I'll note it.